test/framework/options.py

```diff
@@ -2829,6 +2831,9 @@ def run_and_assert(args, msg, words_expected=None, words_unexpected=None):
         not_expected = [testdoval, testdonthdr, testdontval]
         run_and_assert(args, "case E", expected, not_expected)
 
+        # cleanup downloads
+        shutil.rmtree(tmpdir)
```

Comment:
I think this is not required. The test cases creates and removes the parent folder already

Reply:
Indeed, but it doesn't hurt either (this is cleanup of a subdir), so fine